docs/sql-migration-guide.md

```diff
@@ -56,6 +56,8 @@ license: |
   In Spark 3.1 and earlier, table refreshing leaves dependents uncached.
 
   - In Spark 3.2, the usage of `count(tblName.*)` is blocked to avoid producing ambiguous results. Because `count(*)` and `count(tblName.*)` will output differently if there is any null values. To restore the behavior before Spark 3.2, you can set `spark.sql.legacy.allowStarWithSingleTableIdentifierInCount` to `true`.
+ 
+  - In Spark 3.2, a `SHOW TBLPROPERTIES` clause's output shows `key` and `value` columns whether you specify the table property `key` or not. In Spark 3.1 and earlier, a `SHOW TBLPROPERTIES` clause's output only shows a `value` column when you specify the table property `key`.
```

Comment:
can we follow https://github.com/apache/spark/pull/31245/files#diff-b6c16c7a1a4e60bacda4674b52f9eecde3c8a86f281fb3ef21052267f94c1f01R44 ?

Reply:
> can we follow https://github.com/apache/spark/pull/31245/files#diff-b6c16c7a1a4e60bacda4674b52f9eecde3c8a86f281fb3ef21052267f94c1f01R44 ?

updated

Reply:
> can we follow https://github.com/apache/spark/pull/31245/files#diff-b6c16c7a1a4e60bacda4674b52f9eecde3c8a86f281fb3ef21052267f94c1f01R44 ?

How about current?